spec/models/vote_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Vote, type: :model do
+  describe 'relations' do
+    it { should belong_to(:user) }
```

Comment:
Tests for the reverse of these are missing :P

Reply:
👍 done